modules/terraform-aci-interface-state/main.tf

```diff
@@ -0,0 +1,7 @@
+resource "aci_rest_managed" "fabricRsOosPath" {
+  dn         = "uni/fabric/outofsvc/rsoosPath-[topology/pod-${var.pod_id}/paths-${var.node_id}/pathep-[eth${var.module}/${var.port}]]"
```

Comment:
The `dn` line will need to be updated to support FEX and sub-ports.

`fex_id` and `sub_port` could default to null to use as the test whether we're shutting down a port, sub-port, or fex port.